scripts/biomedical/NCBI_Taxonomy/scripts/tests.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Pradeep Kumar Krishnaswamy
+# Date: 18-Apr-2024
+
+#!/bin/bash
+
+# Command
```

Comment:
In addition to the tests being run, please include running the java tool. Please add these additional lines:

# download data commons java test tool version 0.1-alpha.1k
mkdir -p tmp; cd tmp
wget https://github.com/datacommonsorg/import/releases/download/0.1-alpha.1k/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar
cd ..

# run java test tool
java -jar tmp/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar lint tMCFs/ncbi_taxonomy.tmcf ncbi_taxonomy.csv *.mcf output/*.mcf
mv dc_generated ncbi_taxonomy_report